Authenticator/Source/SegmentedControlRow.swift

```diff
@@ -25,6 +25,18 @@
 
 import UIKit
 
+struct SegmentedControlRowModel<Action> {
+    let segments: [(title: String, action: Action)]
+    let selectedSegmentIndex: Int?
+
+    init<V: Equatable>(options: [(title: String, value: V)], value: V, @noescape changeAction: (V) -> Action) {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 111 characters (line_length)